test/distributions/test_constraints.py

```diff
@@ -41,6 +52,15 @@ def build_constraint(constraint_fn, args, is_cuda=False):
     t = torch.cuda.DoubleTensor if is_cuda else torch.DoubleTensor
     return constraint_fn(*(t(x) if isinstance(x, list) else x for x in args))
 
+@pytest.mark.parametrize('constraint_fn, result, value', EXAMPLES)
+@pytest.mark.parametrize('is_cuda', [False,
+                                     pytest.param(True, marks=pytest.mark.skipif(not TEST_CUDA,
+                                                                                 reason='CUDA not found.'))])
+def test_constraint(constraint_fn, result, value, is_cuda):
+    t = torch.cuda.DoubleTensor if is_cuda else torch.DoubleTensor
+    assert constraint_fn.check(t(value)).all() == result, \
+        "Error in checking postive example of {}".format(constraint_fn)
```

Comment:
nit: since we have only a single assert in this test, we can omit the assert message. Pytest will mark the specific instance that fails this check.

Reply:
I didn't noticed that. Thanks for letting me know. I will delete the message.